bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/WZTo2L2Q01j_5f_NLO_FXFX/WZTo2L2Q01j_5f_NLO_FXFX_madspin_card.dat

```diff
@@ -1,13 +1,9 @@
 set ms_dir ./madspingrid
-
 set Nevents_for_max_weigth 250 
 set max_weight_ps_point 400  
 set BW_cut 40                 
```

Comment:
Can you confirm with MadGraph authors that giving 2 different BW cutoffs in MG (bwcutoff=15) and MS (bwcutoff=40) are fine?
I remember him saying it's not physical

Reply:
@sihyunjeon 
I am not very experienced with MG/MS. I will take your suggestion as it is. If it's not physical, I can match them either bwcutoff=15 or 40.

Reply:
Your m(ll from Z) will be (91 - width X bwcutoff) ~ (91 + width X bwcutoff). If you are fine with such changes, I'll just suggest to use bwcutoff=15 for both MG and MS as this is the MG default choice